File: android/app/src/main/java/com/liive/ride/RideProductConstants.kt
package com.liive.ride

internal object RidePricing {
    const val PoolFare = 9.50
    const val PremiumFare = 12.50
    const val ExclusiveFare = 18.00
    const val PoolCostShareCredit = 2.00
    const val TaxMultiplier = 1.0875
    const val CurrencyScale = 100.0
}

internal object RideTripDefaults {
    const val SingleLegEnrouteTitle = "Your driver is arriving"
    const val MultiLegEnrouteTitle = "On leg 2 of 2"
    const val SingleLegEta = "4 min"
    const val MultiLegEta = "3 min"
    const val SingleLegMarkerLabel = "4 min"
    const val MultiLegMarkerLabel = "Leg 2 · 3 min"
    const val TransferStatus = "Transfer at Hayes St complete · 150m walk"
    const val CompletedDuration = "18 min"
    const val CompletedDistance = "5.2 km"
}

File: Packages/RideSharingFeature/Sources/RideSharingFeature/RideProductConstants.swift
import Foundation

enum RidePricing {
    static let poolFare = 9.50
    static let premiumFare = 12.50
    static let exclusiveFare = 18.00
    static let poolCostShareCredit = 2.00
    static let taxMultiplier = 1.0875
    static let currencyScale = 100.0
}

enum RideTripDefaults {
    static let singleLegEnrouteTitle = "Your driver is arriving"
    static let multiLegEnrouteTitle = "On leg 2 of 2"
    static let singleLegETA = "4 min"
    static let multiLegETA = "3 min"
    static let singleLegMarkerLabel = "4 min"
    static let multiLegMarkerLabel = "Leg 2 · 3 min"
    static let transferStatus = "Transfer at Hayes St complete · 150m walk"
    static let completedDuration = "18 min"
    static let completedDistance = "5.2 km"
}
